src/js/components/DataTable/Body.js

```diff
@@ -54,13 +59,37 @@ const Row = memo(
                 adjustedEvent.datum = datum;
                 adjustedEvent.index = index;
                 onClickRow(adjustedEvent);
+                // putting focus on tbody for WCAG guideline
+                tableRef.current.focus();
+              }
+            : undefined
+        }
+        onMouseEnter={
+          onClickRow
+            ? () => {
+                setLastActive(undefined);
```

Comment:
I'm thinking we might want to simplify this and put more logic in the Body than the Row, the Body is the one keeping track of the active state. For example, the Body can manage `lastActive` as it sees `active` change.